versions.props

```diff
@@ -40,4 +39,7 @@ org.openjdk.jmh:* = 1.23
 # dependency-upgrader:OFF
 # Match conjure-java-runtime okhttp version
 com.squareup.okhttp3:* = 3.13.1
+
+# Don't want to impose a major upgrade on people
+com.palantir.conjure.java.runtime:* = 5.10.0
```

Comment:
This is actually consistent with what we've done in tracing-java: 

https://github.com/palantir/tracing-java/blob/develop/versions.props#L2-L3